src/diff-snapshot.js

```diff
@@ -271,10 +271,10 @@ function diffImageToSnapshot(options) {
     result = { added: true };
   } else {
     const receivedSnapshotPath = path.join(receivedDir, `${snapshotIdentifier}${receivedPostfix}.png`);
-    rimraf.sync(receivedSnapshotPath);
+    rimrafSync(receivedSnapshotPath, { glob: true });
```

Comment:
If you're up for it, it would be cool to use `tintglobby` to do the glob and then delete the resulting files with `rmSync(path, { recursive: true, force: true });` from Node.js because rimraf has 27 dependencies, so it'd be nice to use a slimmer solution